deployment/Dockerfile-mpc-tee

```diff
@@ -41,7 +14,7 @@ RUN \
 ENV SSL_CERT_FILE=/etc/ssl/certs/ca-certificates.crt
 
 WORKDIR /app
-COPY --from=builder /app/target/x86_64-unknown-linux-gnu/release/mpc-node mpc-node
+COPY target/reproducible/mpc-node mpc-node
```

Comment:
This looks strange to me. Can you explain why we can copy the `mpc-node` binary from outside the docker environment? Shouldn't we build it when we build the container?

Reply:
because `repro-env` reuses the outside folder, so its result is copied there

Reply:
Alright makes sense. Thanks for the explanation.